app/src/main/assets/special_thanks.csv

```diff
@@ -1,4 +1,6 @@
-221 Pixels;Logo design;https://avatars2.githubusercontent.com/u/58243143?s=60&v=4
-Anxhelo Lushka;Website design;https://avatars2.githubusercontent.com/u/25004151?s=60&v=4
-ByteHamster;Forum admin;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
-Keunes;Communications;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
+ByteHamster;Project lead;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
+Keunes;Project lead;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
+Femmdi;Translations coordinator;https://avatars2.githubusercontent.com/u/47671383?s=60&v=4
+Ryan Gorley (Freehive);Brand design;https://avatars2.githubusercontent.com/u/12849958?s=60&v=4
```

Comment:
I think in the meeting he said we should write `Ryan Gorley @ Freehive`, but I'm not entirely sure

Reply:
I think the important thing is that we have his name and business name.